doc/ref_model/chapters/chapter03.md

```diff
@@ -23,7 +23,7 @@ Although a couple of explicit and implicit abstraction models (e.g. in the conte
 -	**Scope:** the model should describe the most relevant virtualised infrastructure resources (incl. acceleration technologies) an NFVI needs to provide for hosting Telco workloads
 -	**Separation of Concern:** the model should support a clear distinction between the responsibilities related to maintaining the network function virtualisation infrastructure and the responsibilities related to managing the various VNF workloads
 -	**Simplicity:** the amount of different types of resources (including their attributes and relationships amongst one another) should be kept to a minimum to reduce the configuration spectrum which needs to be considered
--	**Declarative:** the model should allow for a declarative description of the required NFVI capabilities for on-boarding and maintaining workloads
+- **Declarative**: the model should allow for the description of the final state of the NFVI resources for automated life cycle management
```

Comment:
It is not only the NFVI resources, but also the NFVI configuration what we would need to describe in a declarative way.

Reply:
That is correct - I have included the configuration aspect into this principle